### ISSUE TITLE
Error when using js_inline_reverse template tag

### ISSUE THREAD

**Issue by anlambert** [ierror/django-js-reverse#83]
In my project, I use the `js_inline_reverse` template tag.

Since the upgrade of `django-js-reverse` to version 0.9, the following error is reported
when Django renders a template containing the template tag above:
```
Django version 1.11.20, using settings 'swh.web.settings.development'
Starting development server at http://127.0.0.1:5004/
Quit the server with CONTROL-C.
Exception while resolving variable 'urlconf' in template 'homepage.html'.
Traceback (most recent call last):
  File "/home/antoine/.local/lib/python3.5/site-packages/django/template/base.py", line 882, in _resolve_lookup
    current = current[bit]
TypeError: 'WSGIRequest' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/antoine/.local/lib/python3.5/site-packages/django/template/base.py", line 890, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'WSGIRequest' object has no attribute 'urlconf'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/antoine/.local/lib/python3.5/site-packages/django/template/base.py", line 896, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'urlconf'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/antoine/.local/lib/python3.5/site-packages/django/template/base.py", line 903, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [urlconf] in "<WSGIRequest: GET '/'>"
```

It seems the issue appeared with this commit: 6de4f62f2dd396ffdaaeb7aaae283b1b29797ec0

**Comment by graingert** [comment]
Hmm that's odd, it does catch that exception...

**Comment by graingert** [comment]
can you create a minimal reproducible example project? The stack trace is extremely short

**Comment by graingert** [comment]
Hmm this looks like it's a debug message:

https://github.com/django/django/blob/1.11.20/django/template/base.py#L921-L926

**Comment by anlambert** [comment]
It is indeed and it is not critical as the template renders correctly.
But it generates a lot of noise when debug mode is activated so this should be fixed.
I will take a deeper look on it tomorrow.

**Comment by graingert** [comment]
I've opened https://github.com/django/django/pull/11404

**Comment by graingert** [comment]
Note that it's no longer recommended to use `js_inline_reverse` as it relies on inherently insecure Inline JavaScript, which will prevent you using `django-csp`

**Comment by anlambert** [comment]
Thanks for your input and the issue analysis. 

Prior to using `js_inline_reverse`, I integrated `django_js_reverse` through a Django view.
But to ease the [LibreJS](https://www.gnu.org/software/librejs/) support for my project, I ended up using the template tag approach (gory details [here](https://forge.softwareheritage.org/rDWAPPSc0614f72f3c522a16c8f8d2f8fc5e087f71ba5c9)).

Nevertheless, you are right, this is not really secure. I managed to get back to the integration with a view I was using before while keeping the LibreJS support. Proceeding this way effectively makes the annoying debug outputs go away.

From my point of view, this issue should be closed.
